### ISSUE TITLE
Can't get pgoapi via pip without installing git

### ISSUE THREAD

**Issue by Azryyal** [scottstamp/PokemonGo-Map#4]
pip install -r requirement.txt

Got this at the end of the install:

Obtaining pgoapi from git+git://github.com/keyphact/pgoapi.git@fd462be412ac347ab517b68f269e2342a7226909#egg=pgoapi (from -r requirements.txt (line 18))
  Cloning git://github.com/keyphact/pgoapi.git (to fd462be412ac347ab517b68f269e2342a7226909) to c:\pokemongo-map-develop\src\pgoapi
  Error [Error 2] The system cannot find the file specified while executing command git clone -q git://github.com/keyphact/pgoapi.git C:\PokemonGo-Map-develop\src\pgoapi
Cannot find command 'git'

I don't have git installed on my laptop. If that is a requirement, then that should be noted someplace. If that shouldn't be a requirement, then the requirement.txt file shouldn't be requesting a clone.

**Comment by skvvv** [comment]
You need to install "git" to be able to clone.
Requirement text is only for python based modules.
git command is something different, not related to python in any way.

**Comment by Azryyal** [comment]
Yeah, I got that. But there should be something, perhaps in the Wiki or README that says this requires git installed. I had no problem figuring out that I needed to install git, but I would have done so first if there was something that told me to do so.

**Comment by MangoTheTango** [comment]
how do i install git, im sorry for sounding really nooby, but i installed git for windows from the website and it doesnt seem to work. i get the same thing  The system cannot find the file specified while executing command git clone -q git://github.com/keyphact/pgoapi.git C:\PokemonGo-Map-develop\src\pgoapi
Cannot find command 'git'. What am i doing wrong?

**Comment by imynwulf** [comment]
@Azryyal So.... You're on git hub and don't expect to have to use git? I'd say that's sort of understood.

**Comment by Azryyal** [comment]
Why? I didn't need it to download AHAAA's version and use it.

**Comment by imynwulf** [comment]
AHAAA used to include an older copy of the pgoapi. At some point, the develop branch was changed to download a new version as part of the pip install. 

Either you had an old version, or you were on the master branch.

**Comment by Azryyal** [comment]
Right. I wasn't interested in developing. I just wanted to use, so I was on the 2.1.0 branch (release). 

Ok. I've updated README.md and created pull request #10 . Once that is pulled in, this can be closed out.
